src/ViewUser.js

```diff
@@ -559,25 +559,33 @@ class ViewUser extends React.Component {
       (
         <PaneMenu>
           {
-            tagsEnabled && <IconButton
-              icon="tag"
-              title={<FormattedMessage id="ui-users.showTags" />}
-              id="clickable-show-tags"
-              onClick={this.props.tagsToggle}
-              badgeCount={tags.length}
-              aria-label={<FormattedMessage id="ui-users.showTags" />}
-            />
+            tagsEnabled &&
+            <FormattedMessage id="ui-users.showTags">
+              { ariaLabel => (
+                <IconButton
+                  icon="tag"
+                  id="clickable-show-tags"
+                  onClick={this.props.tagsToggle}
+                  badgeCount={tags.length}
+                  aria-label={ariaLabel}
```

Comment:
@zburke, IconButton has an ariaLabel prop which than passed as aria-label to the Button.